.github/workflows/archive.yml

```diff
@@ -0,0 +1,28 @@
+name: Create Download on Release
+on: 
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Master
+      uses: actions/checkout@master      
+    - name: Set Tag env
+      run: echo ::set-env name=RELEASE_VERSION::${GITHUB_REF:10}
+    - name: Test tag
+      run: echo ${{ env.RELEASE_VERSION }}
+    - name: Prepare folder
+      run: |
+        mkdir ./PayonePayment
+        cp -r src CHANGELOG* README.md composer.json ./PayonePayment/
+    - name: Build the zip
+      uses: thedoctor0/zip-release@master
```

Comment:
Maybe we should default to a known working release instead of `master`?

Reply:
yep good idea